pageserver/src/tenant/storage_layer/image_layer.rs

```diff
@@ -250,16 +248,20 @@ impl AsLayerDesc for ImageLayer {
     }
 }
 
+#[async_trait::async_trait]
```

Comment:
Would not be needed if we instead downcast all of the assumed deltalayers in compaction, and got rid of the methods here.

Reply:
I think this was done, so resolving.